js-app/package.json

```diff
@@ -1,38 +1,40 @@
 {
   "name": "js-app",
   "version": "1.0.0",
-  "main": "index.js",
   "license": "MIT",
   "description": "Testing application for smart contract wallet",
   "scripts": {
-    "generate-types": "./generate-types.sh",
-    "test": "rm -rf build/ && npx tsc && npx ts-node node_modules/jasmine/bin/jasmine",
-    "prepare": "cd .. && ./js-app/node_modules/.bin/husky install js-app/.husky"
+    "prepare": "cd .. && ./js-app/node_modules/.bin/husky install js-app/.husky",
+    "prettier": "prettier --write **/*.ts",
+    "generate:types": "npm run types:proxy && npm run types:factory && npm run types:govec",
+    "types:proxy": "cosmwasm-typescript-gen generate --schema ../contracts/proxy/schema --out ./types --name Proxy",
+    "types:factory": "cosmwasm-typescript-gen generate --schema ../contracts/factory/schema --out ./types --name Factory",
+    "types:govec": "cosmwasm-typescript-gen generate --schema ../contracts/govec/schema --out ./types --name Govec",
+    "test": "jest --config config/jest.json --runInBand --collectCoverage"
```

Comment:
For local dev, maybe we do not use `--runInBand`, but only in the GHA?

Reply:
or `--ci`?

Reply:
Currently is not possible because if we don't specific --runInBand it just run tests in parallel and I got an error saying that the transaction I'm trying to do it's been processing. I guess it's related with contract instantiation so I think we could remove that flag when we separate the deployment part from the tests.